src/main/java/MessageListener.java

```diff
@@ -1,31 +0,0 @@
-import events.*;
-import it.sauronsoftware.cron4j.Scheduler;
-import net.dv8tion.jda.api.JDA;
-import net.dv8tion.jda.api.JDABuilder;
-import net.dv8tion.jda.api.hooks.ListenerAdapter;
-import util.RemindChecker;
-
-import javax.security.auth.login.LoginException;
-import java.util.TimerTask;
-
-public class MessageListener extends ListenerAdapter {
-    public static void main(String[] args) throws LoginException {
-        //TODO !help command
-        //TODO save remind requests to file
-        JDA api = JDABuilder.createDefault("token").build();
-        api.addEventListener(new Hello());
-        api.addEventListener(new Message());
-        api.addEventListener(new PingPong());
-        api.addEventListener(new Reminder("src\\main\\resources\\reminders.txt"));
```

Comment:
Dziala na Windowsie, nie działa gdzieś indziej.
Jeżeli zamienisz `\\` tym: `File.separator` to możesz przestać się martwić tym czy zadziała na windowsie czy na wszystkich.
Albo, Paths::get (https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html#get(java.lang.String,%20java.lang.String...))